Doc/library/sqlite3.rst

```diff
@@ -2301,6 +2419,10 @@ regardless of the value of :attr:`~Connection.isolation_level`.
    :mod:`!sqlite3` used to implicitly commit an open transaction before DDL
    statements.  This is no longer the case.
 
+.. versionchanged:: 3.12
+   The recommended way of controlling transactions is now via the
+   :attr:`~Connection.autocommit` attribute.
```

Comment:
Since the only "change" here is deprecating this behavior and adding a recommended replacement, the ``deprecated`` directive seems a lot more appropriate here (and is also helpful programmatically).

```suggestion
.. deprecated:: 3.12
   The recommended way of controlling transactions is now via the
   :attr:`~Connection.autocommit` attribute.
```

Furthermore, as discussed, it would be even better to be explicit about current removal plans, e.g.

```suggestion
.. deprecated-removed:: 3.12 3.16
   The recommended way of controlling transactions is now via the
   :attr:`~Connection.autocommit` attribute.
```